packages/backend/src/tests/outgoingPayment.ts

```diff
@@ -115,16 +116,15 @@ export async function createOutgoingPaymentWithReceiver(
     walletAddressId: args.receivingWalletAddress.id
   })
 
-  const connectionService = await deps.use('connectionService')
   const receiver = new Receiver(
-    incomingPayment.toOpenPaymentsType(
-      args.receivingWalletAddress,
-      connectionService.get(incomingPayment)!
+    incomingPayment.toOpenPaymentsTypeWithMethods(
+      args.receivingWalletAddress
     )
```

Comment:
Not essential, since I think this will still need to be changed somehow, but this should be:

```
  const receiver = new Receiver(
    incomingPayment.toOpenPaymentsTypeWithMethods(args.receivingWalletAddress, {
      ilpAddress: 'test.ilp' as IlpAddress,
      sharedSecret: Buffer.from('')
    })
  )
```

Currently it will error because the methods will be an empty array, and receiver needs the payment method.

Reply:
> Not essential

I take that back. Not sure what the ultimately correct way is, but several more tests will fail without this tweak (anything calling `createOutgoingPaymentWithReceiver`).